// plotly.js-locales/index.js
/**
* Copyright 2012-2020, Plotly, Inc.
* All rights reserved.
*
* This source code is licensed under the MIT license found in the
* LICENSE file in the root directory of this source tree.
*/

exports['af'] = require('./af.js');
exports['am'] = require('./am.js');
exports['ar-dz'] = require('./ar-dz.js');
exports['ar-eg'] = require('./ar-eg.js');
exports['ar'] = require('./ar.js');
exports['az'] = require('./az.js');
exports['bg'] = require('./bg.js');
exports['bs'] = require('./bs.js');
exports['ca'] = require('./ca.js');
exports['cs'] = require('./cs.js');
exports['cy'] = require('./cy.js');
exports['da'] = require('./da.js');
exports['de-ch'] = require('./de-ch.js');
exports['de'] = require('./de.js');
exports['el'] = require('./el.js');
exports['eo'] = require('./eo.js');
exports['es-ar'] = require('./es-ar.js');
exports['es-pe'] = require('./es-pe.js');
exports['es'] = require('./es.js');
exports['et'] = require('./et.js');
exports['eu'] = require('./eu.js');
exports['fa'] = require('./fa.js');
exports['fi'] = require('./fi.js');
exports['fo'] = require('./fo.js');
exports['fr-ch'] = require('./fr-ch.js');
exports['fr'] = require('./fr.js');
exports['gl'] = require('./gl.js');
exports['gu'] = require('./gu.js');
exports['he'] = require('./he.js');
exports['hi-in'] = require('./hi-in.js');
exports['hr'] = require('./hr.js');
exports['hu'] = require('./hu.js');
exports['hy'] = require('./hy.js');
exports['id'] = require('./id.js');
exports['is'] = require('./is.js');
exports['it'] = require('./it.js');
exports['ja'] = require('./ja.js');
exports['ka'] = require('./ka.js');
exports['km'] = require('./km.js');
exports['ko'] = require('./ko.js');
exports['lt'] = require('./lt.js');
exports['lv'] = require('./lv.js');
exports['me-me'] = require('./me-me.js');
exports['me'] = require('./me.js');
exports['mk'] = require('./mk.js');
exports['ml'] = require('./ml.js');
exports['ms'] = require('./ms.js');
exports['mt'] = require('./mt.js');
exports['nl-be'] = require('./nl-be.js');
exports['nl'] = require('./nl.js');
exports['no'] = require('./no.js');
exports['pa'] = require('./pa.js');
exports['pl'] = require('./pl.js');
exports['pt-br'] = require('./pt-br.js');
exports['pt-pt'] = require('./pt-pt.js');
exports['rm'] = require('./rm.js');
exports['ro'] = require('./ro.js');
exports['ru'] = require('./ru.js');
exports['sk'] = require('./sk.js');
exports['sl'] = require('./sl.js');
exports['sq'] = require('./sq.js');
exports['sr-sr'] = require('./sr-sr.js');
exports['sr'] = require('./sr.js');
exports['sv'] = require('./sv.js');
exports['sw'] = require('./sw.js');
exports['ta'] = require('./ta.js');
exports['th'] = require('./th.js');
exports['tr'] = require('./tr.js');
exports['tt'] = require('./tt.js');
exports['uk'] = require('./uk.js');
exports['ur'] = require('./ur.js');
exports['vi'] = require('./vi.js');
exports['zh-cn'] = require('./zh-cn.js');
exports['zh-hk'] = require('./zh-hk.js');
exports['zh-tw'] = require('./zh-tw.js');
